CMakeLists.txt

```diff
@@ -218,26 +218,26 @@ include_directories(${PIXMAN_INCLUDE_DIR})
 if(USE_SHARED_FREETYPE)
   find_package(Freetype REQUIRED)
 else()
-  set(FREETYPE_LIBRARY freetype)
-  set(FREETYPE_INCLUDE_DIR ${FREETYPE_DIR}/include)
+  set(FREETYPE_LIBRARIES freetype)
+  set(FREETYPE_INCLUDE_DIRS ${FREETYPE_DIR}/include)
 endif()
-include_directories(${FREETYPE_INCLUDE_DIR})
+include_directories(${FREETYPE_INCLUDE_DIRS})
 
 if(USE_SHARED_GIFLIB)
   find_package(GIF REQUIRED)
 else()
   set(GIF_LIBRARIES giflib)
-  set(GIF_INCLUDE_DIR ${GIFLIB_DIR}/lib)
+  set(GIF_INCLUDE_DIRS ${GIFLIB_DIR}/lib)
 endif()
-include_directories(${GIF_INCLUDE_DIR})
+include_directories(${GIF_INCLUDE_DIRS})
 
 if(USE_SHARED_JPEGLIB)
   find_package(JPEG REQUIRED)
 else()
   set(JPEG_LIBRARIES jpeg)
-  set(JPEG_INCLUDE_DIR ${LIBJPEG_DIR})
+  set(JPEG_INCLUDE_DIRS ${LIBJPEG_DIRS})
```

Comment:
I just saw that this should be `LIBJPEG_DIR` instead of `LIBJPEG_DIRS`, I'll push a fix.